### DISCUSSION TITLE
Can't load library libduckdb_java.so_windows_arm64

### DISCUSSION THREAD

**Discussion by boomah** [duckdb/duckdb#15451] [Q&A]
Occasionally we need to run our Java server on windows on arm. It can't find the arm64 lib for windows in the jar and indeed, it doesn't exist. Only the ones for linux, osx and windows amd64. Any plans to create one for libduckdb_java.so_windows_arm64?
![image](https://github.com/user-attachments/assets/fdf90092-8746-42cb-8445-e147afbfd68e)

**Comment by Tishj** [comment]
You might want to raise this here instead:
https://github.com/duckdb/duckdb-java
